messages.pot

```diff
@@ -287,9 +290,18 @@ msgstr ""
 msgid "comma"
 msgstr ""
 
+msgid "command_not_available_yet_exception"
```

Comment:
I don't really understand why the exceptions also hop around in this PR, did we move them earlier?

Reply:
These exceptions are for the live statistics module! They represent the translations for the exception groups that we discussed a while back for that PR. Maybe it was an error from my part, but `messages.pot` isn't checked in in that PR (#4166) and maybe that is what caused the mess. Here's where I edited them: https://github.com/hedyorg/hedy/pull/4166/files#diff-7d430e5ce7e9bf543efa95783df17746ec4ddcbeb38599d013266447bf075b2c

Reply:
Ah yes, got it!